pkg/controller/account/account_controller.go

```diff
@@ -528,6 +528,33 @@ func (r *ReconcileAccount) nonCCSAssignAccountID(reqLogger logr.Logger, currentA
 		return err
 	}
 
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		reqLogger.Error(err, "failed getting shard name from configmap data")
+		return err
+	}
+
+	hiveName, ok := configMap.Data["shard-name"]
+
+	if ok {
+		// Add tag
+		inputTag := &organizations.TagResourceInput{
+			ResourceId: aws.String(awsAccountID),
+			Tags: []*organizations.Tag{
```

Comment:
Do we also need to add the `claimed: true` tag here as well?

Reply:
we decided that non-ccs accounts will only have their hive name tagged